app/assets/javascripts/sephora_style_guide/slider.js

```diff
@@ -0,0 +1,2447 @@
+// Ion.RangeSlider
```

Comment:
How did you evaluate if this is the right library to use for us? You should document any dependencies this might bring with it. Also a compat table to see if it works nicely on all browsers and devices.

Reply:
I copied it off based on haikal's preferred interaction. As to what dependencies that it bring, I just copy and pasted on the `.js` and `.css` instead of coding it from scratch.